spec/fixtures/data/webserver.log

```diff
@@ -0,0 +1,2 @@
+/ 127.0.0.1
+/search 127.0.0.1
```

Comment:
Small file because it's used only for the gateway test. In all other places, the gateway output is stubbed.